mathics/core/expression.py

```diff
@@ -199,9 +199,9 @@ def __init__(
     ):
         self.options = None
         self.pattern_sequence = False
-        if isinstance(head, str):
-            # We should fix or convert to to_expression all nonSymbol uses.
-            head = Symbol(head)
+        assert isinstance(head, BaseElement)
+        assert isinstance(elements, tuple)
```

Comment:
I have been wavering on this this. When you pass `*x` the result is always a tuple. But I guess it doesn't hurt to check.

Reply:
I put it just to check that in all the platforms and versions, we get the right type. We can comment out these checks now.